lib/json-canonicalize.js

```diff
@@ -0,0 +1,65 @@
+// https://github.com/cyberphone/json-canonicalization
+// https://www.apache.org/licenses/LICENSE-2.0
+/* eslint-disable */
+
+// ES6 based JSON canonicalizer
+'use strict';
+var canonicalize = function(object) {
```

Comment:
Convert `var` to `const`, etc.

Reply:
We could update to our style but I left it as-is thinking it would be easier to diff and update later if needed.  Should we partially or fully resstyle it?

Reply:
Fixed by using `canonicalize` package.